### PR TITLE
trigger: run 'race' and 'deadlock' builds

### PR DESCRIPTION
**Pull request by tamird**
Closes https://github.com/cockroachdb/cockroach/issues/8897.
Closes https://github.com/cockroachdb/cockroach/issues/9713.

### CODE REVIEW

File: cmd/teamcity-trigger/main.go [hunk 1]
```diff
@@ -29,13 +29,16 @@ func main() {
...
+	for _, goflagsVal := range []string{"-race", "-tags deadlock"} {
+		for _, importPath := range importPaths {
+			build, err := client.QueueBuild(*buildTypeID, *branchName, map[string]string{
+				"env.PKG":     importPath,
+				"env.GOFLAGS": goflagsVal,
+			})
+			if err != nil {
+				log.Fatalf("failed to create teamcity build for %s: %s", importPath, err)
+			}
+			log.Printf("created teamcity build for %s: %s", importPath, build)
```
Comment (petermattis) [inline cmd/teamcity-trigger/main.go]: Does this show that a race/deadlock build is being performed? (I'm too lazy to see what `build` contains).

**Review by petermattis — APPROVED** [review]
LGTM

File: cmd/teamcity-trigger/main.go (hunk 1 above)
Comment (tamird) [reply cmd/teamcity-trigger/main.go]: Done.

Comment (petermattis) [reply cmd/teamcity-trigger/main.go]: Ack.

**Pull request merged**